packages/language/src/documentation.ts/pli-documentation-provider.ts

```diff
@@ -0,0 +1,69 @@
+/**
```

Comment:
Is the `.ts` in the folder name `documentation.ts` on purpose?

Reply:
No, just an oversight, let me fix that :D